recipes/xorg/all/conanfile.py

```diff
@@ -23,6 +23,12 @@ def validate(self):
     def package_id(self):
         self.info.clear()
 
+    def requirements(self):
+        # This is necessary to prevent other packages in the dependency tree from using the "libuuid" recipe
+        # which conflicts with the system installation of `util-linux-libuuid` that `xorg` installs. See #17427, #17485
+        # This has no functional bearing on xorg, however, as it links against the system library installed in util-linux-libs
+        self.requires("util-linux-libuuid/2.39")
```

Comment:
```suggestion
        self.requires("util-linux-libuuid/[*]")
```
Could it be set to accept any version, maybe, to avoid the need for version bumps? (the pattern works, I tested it locally)

Reply:
I'm not opposed to this provided it gets maintainer buy in. @uilianries @RubenRBS thoughts?

Reply:
Please check: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges

> Conan maintainers may introduce this for other dependencies over time. Outside of the cases outlined above, version ranges are not allowed in ConanCenter recipes.